src/SFA.DAS.AdminService.Web/Models/Roatp/BooleanConverter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using CsvHelper;
+using CsvHelper.Configuration;
+using CsvHelper.TypeConversion;
+
+namespace SFA.DAS.AdminService.Web.Models.Roatp
+{
+    public class BooleanConverter : DefaultTypeConverter
+    {
+        public override string ConvertToString(object value, IWriterRow row, MemberMapData memberMapData)
+        {
+            if (value == null)
+            {
+                return string.Empty;
+            }
+
+            var boolValue = (bool)value;
+
+            return boolValue ? "Y" : "N";
```

Comment:
how about bool.TryConvert as better solution? If value is empty then it'll return Y which may not be intended

Reply:
A null value returns empty string, but realistically this converter is for use with a bool so not really an issue?